package.json

```diff
@@ -32,7 +32,7 @@
     "moment": "2.29.4",
     "pg": "8.7.3",
     "pg-hstore": "2.3.4",
-    "sequelize": "6.21.2"
+    "sequelize": "^6.29.3"
```

Comment:
Best practise to use exact versions to make our application more predictable. Can add to the npm config to save only exact versions

`npm config set save-exact=true`